resources/aws_ec2.go

```diff
@@ -44,6 +45,12 @@ const (
 	// AwsPrefix is a const which gets prepended onto object names. We can only use
 	// alphanumeric chars, underscores and hyphens for sns topics and cloud watch rules.
 	AwsPrefix = "_mgmt-"
+	// Ec2Prefix is added to the names of sns and cloudwatch objects.
+	Ec2Prefix = AwsPrefix + "ec2-"
+	// SnsPrefix gets prepended onto the sns topic.
+	SnsPrefix = Ec2Prefix + "sns-"
+	// topicName is the name of the sns topic created by snsMakeTopic.
+	topicName = SnsPrefix + "events"
```

Comment:
Should this be private? If it's sns specific can you call it SnsTopicName? or snsTopicName

Reply:
Is there a reason you think it should be public?

Reply:
It makes sense for this to be public if an external non-mgmt tool might want to look up this value to know what key is expected on the aws side. Eg: so that it also doesn't conflict with it for example. If this isn't the case, then it can be private. (I genuinely don't know which is needed.)

Reply:
Fair enough. Best to avoid collisions as much as possible.